pegasus/sites.v3/code.org/public/learn2016/index.haml

```diff
@@ -17,7 +17,7 @@ social:
 -# Not ideal but we can pull the undigested file from /blockly.
 %script{src: CDO.studio_url('/blockly/js/tutorialExplorer.js')}
 
-- tutorials = Tutorials.new(:tutorials)
+- tutorials = Tutorials.new(:tutorials).contents
```

Comment:
Should tutorials be renamed to tutorialsContents so that you don't confuse them?